ci-operator/step-registry/openstack-k8s-operators/deploy/openstack-k8s-operators-deploy-commands.sh

```diff
@@ -174,6 +174,13 @@ patches:
             rbd_store_user=openstack
             rbd_store_pool=images
             store_description=ceph_glance_store
+$(if [[ "${SERVICE_NAME}" == "IRONIC" ]]; then
```

Comment:
we probably can make this part more generic, to accept a parameter with services to be enabled.
Same thing for additional configuration.
We can work on that in a another PR

Reply:
+1 yeah, there are some more services which are optional so it would make sense to make this part more generic.